labs-ios-starter/Model/Apollo/Web Auth/UserImage.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  UserImage.swift
+//  labs-ios-starter
+//
+//  Created by Kenny on 9/8/20.
+//  Copyright © 2020 Lambda, Inc. All rights reserved.
+//
+
+import Foundation
+
+struct UserImage: Codable {
+    var id: UUID
+    var userId: Member
+    var image: Data
```

Comment:
should we persist a specific image type like UIImage here?

Reply:
Manual codable would allow us to do that, and it'd be cleaner in some regards, or I could implement a convenience method that would return that as an image. Lots of options... Do you think this is good for now? It'll allow us to interact with the API and get an image back